### PR TITLE
Update the support for lazydesigners/dimple

### PR DESCRIPTION
**Pull request by jackytrabbit**
Update the support for lazydesigners/dimple

## Description

Added more layout support.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by jackytrabbit** [comment]
@fauxpark I am sorry that I messed up the previous PR. I tried to resolve the conflicts with Github Desktop and command line but the branch became more complex with new commits. At last I couldn't resolve the conflicts and get more issues. So I opened a new branch  and use the files that are updated in the last PR. Thanks for your help.

And thanks for helping me with this @drashna @tominabox1 @Erovia

**Comment by jackytrabbit** [comment]
And here is the previous PR which is closed
https://github.com/qmk/qmk_firmware/pull/14168

**Comment by jackytrabbit** [comment]
Hi, is there anyone could help me on this? It seems everything is ready.

**Comment by jackytrabbit** [comment]
@fauxpark Don't mean to rush you but do you think this PR is ready for merge? I really appreciate all your works. Let me know if anything is needed. Thank you so much!

File: keyboards/lazydesigners/dimple/rules.mk [hunk 1]
```diff
@@ -8,12 +8,19 @@ BOOTLOADER = atmel-dfu
...
-RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
```
Comment (fauxpark) [inline keyboards/lazydesigners/dimple/rules.mk:19]:
```suggestion
NKRO_ENABLE = yes           # Enable N-Key Rollover
```

File: keyboards/lazydesigners/dimple/rules.mk [hunk 2]
```diff
@@ -8,12 +8,19 @@ BOOTLOADER = atmel-dfu
 #   change yes to no to disable
 #
 BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
-MOUSEKEY_ENABLE = yes        # Mouse keys
-EXTRAKEY_ENABLE = yes        # Audio control and System control
-NKRO_ENABLE = yes            # USB Nkey Rollover
-BACKLIGHT_ENABLE = no        # Enable keyboard backlight functionality
-RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+
```
Comment (fauxpark) [inline keyboards/lazydesigners/dimple/rules.mk:11]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by jackytrabbit** [comment]
@fauxpark Happy New Year! I wonder that is there anything I can do to make this PR ready for merge? Thank you so much for the rewiew.

**Review by fauxpark — CHANGES_REQUESTED** [review]
There is a merge conflict that needs to be resolved, otherwise looks OK.

**Comment by jackytrabbit** [comment]
> There is a merge conflict that needs to be resolved, otherwise looks OK.
 
Oh, I see. The conflict has been resolved. Thank you for your help!

**Comment by jackytrabbit** [comment]
> There is a merge conflict that needs to be resolved, otherwise looks OK.

@fauxpark Thank you so much! Is it OK to merge by now? I know you are busy. Thanks again for your service!

**Comment by jackytrabbit** [comment]
@fauxpark Sorry for boring you again. Just wondering if this PR is OK to be merged. Thanks for what you are doing for community.

**Review by Erovia — APPROVED** [review]

**Pull request merged**